ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDYesNoOption.java

```diff
@@ -1,15 +1,18 @@
 package uk.gov.hmcts.cmc.ccd.domain;
 
+import com.fasterxml.jackson.annotation.JsonValue;
+
 public enum CCDYesNoOption {
-    YES("yes"),
-    NO("no");
+    YES("Yes"),
```

Comment:
we might need to clean data in CCD, Also we use the YesNoOption in the domain to getValue based on the value. Please check if that affects too.